spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/configuration.adoc

```diff
@@ -377,31 +377,184 @@ Request to clean up resources for task execution 1 has been submitted
 
 == Scheduling
 
-Scheduling of tasks is enabled by default in the Spring Cloud Data Flow Kubernetes Server. The following environment variables may be declared as part of the container env section of the server deployment YAML file to customize the defaults if needed:
+This section covers customization of how scheduled tasks are configured. Scheduling of tasks is enabled by default in the Spring Cloud Data Flow Kubernetes Server. Properties are used to influence settings for scheduled tasks and can be configured on a global or per-schedule basis.
```

Comment:
Replace influence with `establish the`

Reply:
keeping as that was done in a prior editing pass